crates/ethereum/cli/Cargo.toml

```diff
@@ -11,13 +11,25 @@ repository.workspace = true
 workspace = true
 
 [dependencies]
-# reth
+# reth dependencies
 reth-cli.workspace = true
 reth-chainspec.workspace = true
+reth-cli-commands.workspace = true
```

Comment:
okay turns out we have a circular dependency here now because we use:

https://github.com/paradigmxyz/reth/blob/main/crates/cli/commands/src/node.rs#L9-L9

I think we can work around this by either
a) remove the default usage of `EthereumChainSpecParser` from reth-cli-commands
b) move `EthereumChainSpecParser` into `reth-cli-commands`

I think we can go with option a)

do you wan to take this prep as a preliminary pr first before we can finalize this @Haxry ?